content/rule_book/internationals.md

```diff
@@ -56,5 +56,5 @@ These rules apply to the Internationals quiz meet conducted once yearly, adminis
 ## Equipment
 
 - Electronic jump-seat equipment must be used for all competitions. A back-up set must be on hand at all times.
-- An unofficial scoreboard must be maintained in full view of the audience at all times, or the score read at the beginning of each time out.
+- An unofficial scoreboard must be maintained in full view of the audience at all times, or the score read at the beginning of each time-out.
```

Comment:
I'd go with "timeout", which [dictionary.com](https://www.dictionary.com/browse/timeout) claims is a word.

Reply:
okay. Then we have other fixes. 
![Screen Shot 2020-11-17 at 9 15 19 PM](https://user-images.githubusercontent.com/13351944/99474126-071a8880-291a-11eb-9940-ee99654da7a1.png)

Reply:
Yup. Looks like. It's a minor nit-pick. I don't particularly care, as long as it's consistent.

Reply:
I chose "timeout"